mathesar_ui/src/component-library/button/Button.scss

```diff
@@ -31,155 +31,225 @@
+        --button-border-color: var(--border-control-hover);
+        --button-shadow: var(--border-shadow) 0 1px 2px 0;
       }
     }
   }
 
   &.btn-secondary {
-    --button-background: linear-gradient(
-      to bottom right,
-      var(--neutral-50),
-      var(--neutral-100)
-    );
-    --button-border-color: var(--neutral-400);
-    --button-color: var(--text-color-primary);
+    background-color: var(--color-navigation-20);
+    --button-border: 0.5px var(--border-control);
+    --button-color: var(--text-control);
     box-shadow:
-      0 1px 2px rgba(0, 0, 0, 0.05),
-      0 1px 3px rgba(0, 0, 0, 0.1),
-      0 1px 2px -1px rgba(0, 0, 0, 0.1);
+      0 1px 2px color-mix(in srgb, var(--border-shadow), transparent 75%),
+      0 1px 3px color-mix(in srgb, var(--border-shadow), transparent 80%),
+      0 1px 2px -1px color-mix(in srgb, var(--border-shadow), transparent 80%);
+
+    &:not(:disabled):not(.page) {
+      &.focus,
+      &:focus {
+        --button-outline: 2px solid var(--border-control-active);
+        outline-offset: 1px;
+      }
+
+      &:hover,
+      &.hover {
+        background-color: var(--color-navigation-hover-25);
+        --button-border-color: var(--border-control-hover);
+        box-shadow:
+          0 4px 6px -1px color-mix(in srgb, var(--border-shadow), transparent
+                10%),
+          0 2px 4px -2px color-mix(in srgb, var(--border-shadow), transparent
+                10%),
+          0 0 0 1px color-mix(in srgb, var(--border-shadow), transparent 10%)
+            inset;
+      }
+
+      &:active,
+      &.active {
+        background-color: var(--color-navigation-active-25);
+        --button-border-color: var(--border-control-active);
+        box-shadow:
+          0 1px 2px color-mix(in srgb, var(--border-shadow), transparent 5%),
+          0 1px 3px color-mix(in srgb, var(--border-shadow), transparent 10%),
+          0 1px 2px -1px color-mix(in srgb, var(--border-shadow), transparent
+                10%);
+      }
+    }
+  }
+
+  &.btn-option-card {
+    background-color: var(--surface-elevated-high);
+    --button-border-color: var(--border-elevated-mid);
+    --button-color: var(--text-control);
+    box-shadow:
+      0 1px 2px color-mix(in srgb, var(--border-shadow), transparent 5%),
+      0 1px 3px color-mix(in srgb, var(--border-shadow), transparent 10%),
+      0 1px 2px -1px color-mix(in srgb, var(--border-shadow), transparent 10%);
 
     &:not(:disabled) {
       &.focus,
       &:focus {
-        --button-outline: 2px solid rgba(189, 193, 198, 0.3);
+        --button-outline: 2px solid var(--color-navigation-auxiliary-40);
         outline-offset: 1px;
       }
 
       &:hover,
       &.hover {
-        --button-background: linear-gradient(
-          to bottom right,
-          var(--neutral-200),
-          var(--neutral-300)
-        );
-        --button-border-color: var(--neutral-500);
+        background-color: var(--color-navigation-auxiliary-hover-10);
```

Comment:
@pavish I had left this as `color-database` because these only show up in the DB connection part of things. I would consider making then `color-primary` rather than introducing a new color here.

Reply:
I agree. I added this here to move the product specific utility colors to App.svelte and to tackle the colors as part of my further review.

I commented below to describe the further steps we require.